op-node/rollup/derive/fuzz_parsers_test.go

```diff
@@ -107,7 +135,7 @@ func FuzzL1InfoAgainstContract(f *testing.F) {
 
 		// Check that our encoder produces the same value and that we
 		// can decode the contract values exactly
-		enc, err := expected.MarshalBinary()
+		enc, err := expected.marshalBinaryPreEcotone()
```

Comment:
A better naming scheme is `marshalBinaryBedrock` and then `marshalBinaryEcotone`. We should standardize to this sort of naming around the codebase. Then adding a new version is very straight forward

Reply:
make sense, done!